createwindow/window_linux.go

```diff
@@ -25,3 +26,28 @@ func ChromeCommand(layout Layout, userdatadir, display string, debuggingPort int
 	}
 	return cmd
 }
+
+func DisplayError(err error, cmdErrorHandler func(err error) error, isFatal bool) error {
+	var messageType string
+	if isFatal {
+		messageType = "error"
+	} else {
+		messageType = "warning"
+	}
+
+	zenity := command.ExternalCommand{
+		Path: "zenity",
+		Arg: []string{
+			fmt.Sprintf("--%s", messageType),
+			"--title",
```

Comment:
nit: Combine the arguments that belong together in one line (--title and its parameter).